tgmc.dme

```diff
@@ -116,13 +118,21 @@
 #include "code\__HELPERS\sorts\MergeSort.dm"
 #include "code\__HELPERS\sorts\TimSort.dm"
 #include "code\_dependencies\components\project.dm"
+#include "code\_dependencies\components\code\__DEFINES\components.dm"
```

Comment:
you'll need to fix these merge conflicts

Reply:
Did that